app/views/mucr_options.scala.html

```diff
@@ -50,7 +50,7 @@
                 legend = Some(Legend(
                     content = Text(messages("mucrOptions.title")),
                     isPageHeading = true,
-                    classes = "govuk-fieldset__legend--xl"
+                    classes = "govuk-fieldset__legend--l"
```

Comment:
This class change caused the header for this page (page 1 of associate) to be too big (compared to other pages in the app)